vulture.py

```diff
@@ -165,8 +167,14 @@ def _get_modules(self, paths, toplevel=True):
                 sys.exit('Error: %s could not be found.' % path)
         return modules
 
+    def _get_stdlib_whitelist_file(self):
+        script = os.path.abspath(stdlib.__file__)
```

Comment:
I think we need to change the way vulture is installed. Currently, only the main script is installed. Since no directory is created for vulture, we can't add the stdlib.py file anywhere. I'm not sure what the best solution is to this problem. I'll do some reading, maybe you have an idea?

https://docs.python.org/3/distutils/setupscript.html#installing-package-data
https://docs.python.org/3/library/pkgutil.html#pkgutil.get_data
https://stackoverflow.com/a/5899643